dvclive/metrics.py

```diff
@@ -41,6 +41,13 @@ def __init__(
                 self._step = step
         else:
             shutil.rmtree(self.dir, ignore_errors=True)
+
+            try:
+                os.remove(self.summary_path)
+                os.remove(self.html_path)
+            except OSError:
```

Comment:
Maybe lets check if summary and html_path exists before deleting?
OSError might happen due to various reasons.

Reply:
What if we replace `OSError` with a more specific `FileNotFoundError`??

I used the `try`/`except` insted of `if` check because that's how it is internally handled in the standard library `pathlib`:

https://github.com/python/cpython/blob/main/Lib/pathlib.py#L1188

Reply:
Got it, no strong preference from me here, yet still: what if `summary_path` does not exist and `html_path` does? In this case we won't delete `html_path`.

Reply:
That's true. I will split the removal